draft-ietf-intarea-proxy-config.md

```diff
@@ -164,6 +164,7 @@ use of a proxy.
 | proxy | No | String containing the URI template or hostname and port of the proxy, depending on the format defined by the protocol | String | "https://proxy.example.org:4443/masque{?target_host,target_port}" |
 | alpn | Yes | An array of Application-Layer Protocol Negotiation protocol identifiers | Array of Strings | ["h3","h2"] |
 | mandatory | Yes | An array of optional keys that client must understand and process to use this proxy | Array of Strings | ["match"] |
+| identifier | No | A string used to refer to the proxy, which can be referenced by other dictionaries | String | "udp-proxy" |
```

Comment:
is the "identifier" key optional or not? Here it is marked as not optional, but the description says it is optional.

Reply:
```suggestion
| identifier | No | A string used to refer to the proxy, which can be referenced by other dictionaries, such as `proxy-match`  | String | "udp-proxy" |
```